src/components/InputProblem.tsx

```diff
@@ -67,14 +68,35 @@ export const InputProblem = ({
     }
   }, [feedback])
 
-  const evaluateInput = (input: string, answer: string): boolean => {
+  const evaluateInput = (input: string | undefined, answer: string): boolean => {
```

Comment:
I'm not sure why this signature needs to be changed. I don't think `input` will ever be `undefined` :man_shrugging: .